app/validators/withdrawn/ecf_validator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Withdrawn
+  class ECFValidator < BaseValidator
+    class << self
+      def reasons
+        [
+          "Left teaching profession",
+          "Moved school",
+          "Mentor no longer being mentor",
+          "School left FIP",
+          "Career break",
+          "Passed Induction",
+          "Other",
+        ].freeze
```

Comment:
We can change it later, but if it is an enum, then do we want to receive it in a human readable form as opposed to, say `passed_induction`?